src/modules/esl-carousel/plugin/touch/esl-carousel.touch.mixin.ts

```diff
@@ -67,7 +92,7 @@ export class ESLCarouselTouchMixin extends ESLCarouselPlugin {
   protected _onPointerMove(event: TouchEvent | PointerEvent | MouseEvent): void {
     if (!this.isTouchStarted) return;
     const point = getTouchPoint(event);
-    const offset = point.x - this.startPoint.x;
+    const offset = this.$host.config.vertical ? point.y - this.startPoint.y : point.x - this.startPoint.x;
```

Comment:
It looks like a separate method. It will also save us from duplication.

```suggestion
protected _getPointerOffset(event: TouchEvent | PointerEvent | MouseEvent): number {
    const point = getTouchPoint(event);
    const {startPoint} = this;
    return this.$host.config.vertical ? point.y - startPoint.y : point.x - startPoint.x;
}
```